wrapper/src/test/java/integration/util/ContainerHelper.java

```diff
@@ -222,6 +222,9 @@ public T withFixedExposedPort(int hostPort, int containerPort) {
                     builder
                         .from(testContainerImageName)
                         .run("mkdir", "app")
+                        .run(testContainerImageName.contains("graalvm")
+                            ? "microdnf install findutils"
```

Comment:
Why do we need this findutils?

Reply:
The GraalVM tests were failing with a `xargs is not available` error. Adding this package fixes that.